### ISSUE TITLE
Oracle: CV06 rule incorrectly converts Oracle's / delimiter to an extra semicolon

### ISSUE THREAD

**Issue by saulotoledo** [sqlfluff/sqlfluff#7053]
### Search before asking

- [x] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.

### What Happened

The CV06 rule incorrectly converts Oracle's `/` statement terminator to an extra `;` when there's a newline between `END;` and `/`. This affects all Oracle PL/SQL blocks.

### Expected Behaviour

Oracle PL/SQL blocks should maintain their proper delimiter format:
```sql
END;
/
```

The `/` delimiter should be preserved and not converted to additional semicolons.

### Observed Behaviour

The CV06 rule converts the Oracle `/` delimiter to an extra semicolon, resulting in:
```sql
END;;
```

This creates invalid Oracle syntax, as the `/` delimiter is required for PL/SQL block termination in Oracle.

### How to reproduce

1. Create an Oracle PL/SQL file (e.g., `oracle_block.sql`) with the following content:

```sql
DECLARE
    my_var VARCHAR2(100);
BEGIN
    my_var := 'Hello World';
    DBMS_OUTPUT.PUT_LINE(my_var);
END;
/
```

2. Configure sqlfluff for Oracle dialect:

```yaml
[sqlfluff]
dialect = oracle

[sqlfluff:rules:convention.terminator]
require_final_semicolon = true
```

3. Install sqlfluff and run it on the file:

```sh
pip install --no-cache-dir sqlfluff==3.4.2
sqlfluff fix /workspace/migrations/example_migration.sql
```

4. Observe that the final line becomes:

```sql
END;;
```

The `/` delimiter is incorrectly replaced with an additional semicolon `;;`.

### Dialect

oracle

### Version

3.4.2

### Configuration

```yaml
[sqlfluff]
dialect = oracle

[sqlfluff:rules:convention.terminator]
require_final_semicolon = true
```

### Are you willing to work on and submit a PR to address the issue?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)
